subsys/bluetooth/host/Kconfig.l2cap

```diff
@@ -56,6 +56,13 @@ config BT_L2CAP_DYNAMIC_CHANNEL
 	  This option enables support for LE Connection oriented Channels,
 	  allowing the creation of dynamic L2CAP Channels.
 
+config BT_L2CAP_DYNAMIC_CHANNEL_ECRED
```

Comment:
Perhaps BT_L2CAP_ECRED would be better here.

Reply:
Updated.